spec/game_session_spec.rb

```diff
@@ -5,8 +5,8 @@
     @player1 = Player.new(" X")
     @player2 = Player.new(" O")
     @display1 = Display.new
-    @rules = Rules.new
-    @game_session = GameSession.new(@player1, @player2, @display1, @rules)
+    @outcome = Outcome.new
+    @game_session = GameSession.new(@player1, @player2, @display1, @outcome)
 
     # @input = StringIO.new
```

Comment:
Why is there commented out code?

Reply:
This commented out code was an earlier attempt to write tests for Game Session before I created the Initializer class to create the Board. I'll be rewriting the Game Session tests and removing these irrelevant comments.

Reply:
This needs to still be addressed (#2).